docs/streams/upgrade-guide.html

```diff
@@ -72,6 +72,57 @@ <h1>Upgrade Guide and API Changes</h1>
         More details about the new config <code>StreamsConfig#TOPOLOGY_OPTIMIZATION</code> can be found in <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-295%3A+Add+Streams+Configuration+Allowing+for+Optional+Topology+Optimization">KIP-295</a>.
     </p>
 
+    <h3><a id="streams_api_changes_240" href="#streams_api_changes_240">Streams API changes in 2.4.0</a></h3>
+
+    <!-- Placeholder KIP-213 -->
+
+    <!-- Placeholder KIP-307 -->
+    <!-- Placeholder KIP-479 -->
+
+    <!-- Placeholder KIP-429 -->
+
+    <p>
+        The 2.4.0 release contains newly added and reworked metrics.
+        <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-470%3A+TopologyTestDriver+test+input+and+output+usability+improvements">KIP-444</a>
+        adds new <emph>client level</emph> (ie, <code>KafkaStreams</code> instance level) metrics, to the existing
```

Comment:
nit: remove comma after metrics (I'd maybe also remove after the "ie" ..?)

Reply:
Also, I `<emph>`think`</emph>` the tag (or whatever it's called) is actually `<em>`  not `<emph>` ...but I also know next to nothing about html 😄